Calibration/HcalCalibAlgos/test/RecAnalyzerHF.cc

```diff
@@ -42,16 +42,16 @@ class RecAnalyzerHF : public edm::one::EDAnalyzer<edm::one::WatchRuns,edm::one::
 
 public:
   explicit RecAnalyzerHF(const edm::ParameterSet&);
-  ~RecAnalyzerHF();
+  ~RecAnalyzerHF() override;
 
   static void fillDescriptions(edm::ConfigurationDescriptions& descriptions);
 
 private:
-  virtual void analyze(edm::Event const&, edm::EventSetup const&) override;
-  virtual void beginJob() override;
-  virtual void endJob() override;
-  virtual void beginRun(edm::Run const&, edm::EventSetup const&) override {}
-  virtual void endRun(edm::Run const&, edm::EventSetup const&) override {}
+  void analyze(edm::Event const&, edm::EventSetup const&) override;
+  void beginJob() override;
+  void endJob() override;
+  void beginRun(edm::Run const&, edm::EventSetup const&) override {}
+  void endRun(edm::Run const&, edm::EventSetup const&) override {}
```

Comment:
I suggest to remove the template argument `edm::one::WatchRuns` in your analyzer and to drop those two methods as they are anyway not used.